shardingsphere-features/shardingsphere-readwrite-splitting/shardingsphere-readwrite-splitting-api/src/main/java/org/apache/shardingsphere/readwritesplitting/api/rule/ReadWriteSplittingDataSourceRuleTest.java

```diff
@@ -0,0 +1,56 @@
+package org.apache.shardingsphere.readwritesplitting.api.rule;
+
+import org.junit.Before;
+import org.junit.Test;
+import java.util.Arrays;
+import java.util.Properties;
+
+import lombok.Getter;
+import lombok.RequiredArgsConstructor;
+
+import static org.hamcrest.CoreMatchers.is;
+import static org.junit.Assert.assertThat;
+
+@RequiredArgsConstructor
+@Getter
+public class ReadWriteSplittingDataSourceRuleTest {
+
+    private ReadwriteSplittingDataSourceRuleConfiguration readwriteSplittingDataSourceRuleConfig;
+    private ReadwriteSplittingDataSourceRuleConfiguration readwriteSplittingDataSourceRuleConfigDynamic;
+
+    @Before
+    public void setup(){
+        readwriteSplittingDataSourceRuleConfig=new ReadwriteSplittingDataSourceRuleConfiguration("ds", "Static", getProperties("write_ds", "read_ds_0,read_ds_1"),"");
+        readwriteSplittingDataSourceRuleConfigDynamic=new ReadwriteSplittingDataSourceRuleConfiguration("ds", "Dynamic", getProperties("write_ds", "read_ds_0,read_ds_1"),"");
+    }
+
+    @Test
+    public void assertGetAutoAwareDataSourceName(){
+
+        ReadwriteSplittingDataSourceRuleConfiguration anotherInstance=new ReadwriteSplittingDataSourceRuleConfiguration("ds","Dynamic",getProperties("write_ds","read_ds_0,read_ds_1"),"");
+        String actual= readwriteSplittingDataSourceRuleConfigDynamic.getProps().getProperty("auto-aware-data-source-name");
+        String testing= anotherInstance.getProps().getProperty("auto-aware-data-source-name");
+        assertThat(testing,is(actual));
+    }
+
+    @Test
+    public void assertGetWriteDataSourceName(){
+        ReadwriteSplittingDataSourceRuleConfiguration anotherInstance=new ReadwriteSplittingDataSourceRuleConfiguration("ds","Static",getProperties("write_ds","read_ds_0,read_ds_1"),"");
+        String actual= readwriteSplittingDataSourceRuleConfig.getProps().getProperty("write-data-source-name");
+        String testing= anotherInstance.getProps().getProperty("write-data-source-name");
+        assertThat(testing,is(actual));
```

Comment:
The expected value does not have to be obtained by creating an object, it can be like this: `assertThat(actual, is("xxx"));`